res/mobileclient.yaml.tmpl

```diff
@@ -0,0 +1,10 @@
+apiVersion: mobile.k8s.io/v1alpha1
```

Comment:
The APIVersion has been changed in the lasted version of MDC. This should be `mdc.aerogear.org/v1alpha1` now.